.gitignore

```diff
@@ -0,0 +1,93 @@
+
+# Created by https://www.gitignore.io/api/intellij
+# Edit at https://www.gitignore.io/?templates=intellij
+
+### Intellij ###
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and WebStorm
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# User-specific stuff
+.idea/**/workspace.xml
```

Comment:
please update to have all .idea/ package ignored, remove obsolete ones

Reply:
Fixed

Reply:
As time as you have included .idea/ -> This stuff can be removed, because it is covered by .idea/

Reply:
Fixed